var/spack/repos/builtin/packages/llvm-openmp/package.py

```diff
@@ -76,10 +93,31 @@ def patch(self):
             os.rename(cmake_mod_dir, os.path.join(self.stage.path, "cmake"))
 
     def cmake_args(self):
-        cmake_args = []
+
+        # Disable LIBOMP_INSTALL_ALIASES, otherwise the library is installed as
+        # libgomp alias which can conflict with GCC's libgomp.
+        cmake_args = ["-DLIBOMP_INSTALL_ALIASES=OFF"]
+        cmake_args.append("-DOPENMP_STANDALONE_BUILD=1")
```

Comment:
This may only be set inside the standalone variant, not always:
```suggestion
```
See below, I think LIBOMP_STANDALONE_BUILD should be replaced by this variable inside the standalone variant.

Reply:
I didn't see the second url - I will change LIBOMP_STANDALONE_BUILD to OPENMP_STANDALONE_BUILD and checkin the next set of updates.   Thanks!